<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QSetupDialogClass</class>
 <widget class="QDialog" name="QSetupDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QSetupDialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>63</y>
     <width>104</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Time for erase:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>18</y>
     <width>104</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Time for reboot:&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>121</x>
     <y>19</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Time for wait reboot ats</string>
   </property>
   <property name="accessibleName" >
    <string>MMM</string>
   </property>
   <property name="inputMask" >
    <string>9999999; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>108</y>
     <width>104</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">T&lt;span style=" font-size:8pt;">ime for send block:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>110</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Time wait for erase flash</string>
   </property>
   <property name="inputMask" >
    <string>9999999; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>60</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Time for wait send block</string>
   </property>
   <property name="inputMask" >
    <string>9999999; </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSetterminalFont" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Select Terminal Font</string>
   </property>
   <property name="text" >
    <string>Select Terminal Font</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>SelectBkColor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushMainColor" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>SelectMnClr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushOk" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>270</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCancel" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>270</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStyle" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>240</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Style:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboStyle" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>240</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Style window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSelectErrorColor" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>200</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>SelectErrColor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSelectCallColor" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>200</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>SelectCallColor</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
